WORKSPACE

```diff
@@ -185,16 +179,9 @@ OVERRIDES = {
     "com.fasterxml.jackson.core:jackson-databind": "2.9.10.1",
 
     "io.netty:netty-all": "4.1.38.Final",
-    "io.netty:netty-buffer": "4.1.38.Final",
-    "io.netty:netty-codec": "4.1.38.Final",
     "io.netty:netty-codec-http2": "4.1.38.Final",
-    "io.netty:netty-codec-http": "4.1.38.Final",
-    "io.netty:netty-codec-socks": "4.1.38.Final",
-    "io.netty:netty-common": "4.1.38.Final",
     "io.netty:netty-handler": "4.1.38.Final",
     "io.netty:netty-handler-proxy": "4.1.38.Final",
-    "io.netty:netty-resolver": "4.1.38.Final",
-    "io.netty:netty-transport": "4.1.38.Final",
```

Comment:
We should keep these as they "pin" the version of all the dependencies that we have right now. Otherwise, they will become "accidental".

Reply:
will be completed in the release prep PR